src/Build/Evaluation/Expander.cs

```diff
@@ -4242,6 +4250,44 @@ private static bool TryGetArg(object[] args, out string arg0)
                 return arg0 != null;
             }
 
+            private static readonly Dictionary<string, StringComparison> StringComparisonNames =
+                new Dictionary<string, StringComparison>(StringComparer.Ordinal)
+                {
+                    {"System.StringComparison.OrdinalIgnoreCase", StringComparison.OrdinalIgnoreCase},
+                    {"System.StringComparison.Ordinal", StringComparison.Ordinal},
+                    {"System.StringComparison.CurrentCulture", StringComparison.CurrentCulture},
+                    {"System.StringComparison.CurrentCultureIgnoreCase", StringComparison.CurrentCultureIgnoreCase},
+                    {"System.StringComparison.InvariantCulture", StringComparison.InvariantCulture},
+                    {"System.StringComparison.InvariantCultureIgnoreCase", StringComparison.InvariantCultureIgnoreCase},
+                    {"OrdinalIgnoreCase", StringComparison.OrdinalIgnoreCase},
```

Comment:
Is it legal to do, e.g. `StringComparison.OrdinalIgnoreCase`?

Reply:
Or see/abstract https://github.com/microsoft/msbuild/pull/5342/files#diff-dee96bf621e4ed93f0db318ca71d4d35R3572